ci/tests/vmachine/vmachine_test.py

```diff
@@ -36,8 +39,10 @@
 
 NUMBER_OF_DISKS = 10
 GRID_IP = test_config.get('main', 'grid_ip')
-TIMEOUT = 30
+TIMEOUT = 70
 TIMER_STEP = 5
+TEMPLATE_DISK_NAME = "templateDisk"
```

Comment:
We should try to get rid of this kind of code
These should either go in a class or in a config file